### ISSUE TITLE
Check mount options of steam folder at startup

### ISSUE THREAD

**Issue by pierinz** [ValveSoftware/steam-for-linux#143]
Steam should check if the partition/disk where .steam/steam/ resides has "exec" mount option.
External devices (and NTFS partitions) are usually mounted with noexec, so even if bin_steam.sh has +x, it can't be executed (and /usr/bin/steam tries to download again, install again and finally dies with an obscure error message).

I know it's a bit tricky, but something like:
grep `df . | tail -n 1 | cut -d' ' -f 1` /etc/mtab | egrep '(user|users|noexec)'
should be a good starting point.

**Comment by eiglow** [comment]
Has this issue been resolved?

**Comment by Naltarunir** [comment]
> Has this issue been resolved?

Nope. This is still a problem and if the exec flag is missing, this does not show up in debug mode over CLI either. 
Currently the steam client is unaware of the flag.
